packages/core/test/types.test.ts

```diff
@@ -955,3 +957,38 @@ describe('actions', () => {
     });
   });
 });
+
+describe('tags', () => {
```

Comment:
I would move those to [`typeHelpers.test.ts`](https://github.com/statelyai/xstate/blob/4e6b54ad50770afa125d3cc51fae66b63c29c31a/packages/core/test/typeHelpers.test.ts#L346) since this file has the most tests related to those helpers